src/utils/remoteVersions.ts

```diff
@@ -0,0 +1,24 @@
+interface MetaPackage {
+  formatVersion: int;
+  name: string;
+  recommended: string[];
+  uid: string;
+}
+
+interface SimplifiedGHReleases {
+  tag_name: string;
+}
+
+// TODO: caching
+export async function getLatestMinecraftVersion(): Promise<string> {
+  const f = await fetch('https://meta.polymc.org/v1/net.minecraft/package.json');
+  const package = await f.json() as MetaPackage;
+  return package.recommended[0];
+}
+
+// TODO: caching
+export async function getLatestPolyMCVersion(): Promise<string> {
```

Comment:
Ok why remove caching it was there

Reply:
because it sucked, lol.

There is probably a good caching library out there, but I dunno which one we could use.

Reply:
Idk why we need a lib for 2 request but I guess I’ll try to find one then